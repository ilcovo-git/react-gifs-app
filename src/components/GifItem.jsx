import PropType from 'prop-types'

export const GifItem = ( {title, url} ) => {
  return (
    <div className="card">
        <img src={ url } alt={ title } />
        <p>{ title }</p>
    </div>
  )
}

GifItem.protoType = {
  title: PropType.string.isRequired,
  url: PropType.string.isRequired
}
