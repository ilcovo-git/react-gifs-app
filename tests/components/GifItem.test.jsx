import { render } from '@testing-library/react';
import { GifItem } from '../../src/components/GifItem';

describe('Evalua Componenet <GifItem />', () => { 

    const title = 'La imagen m√°s bonita';
    const url   = 'https://postman';

    
    test('should be match prop title and url at <GifItem /> component with snapshot', () => { 
        
        const { container} = render(<GifItem title={ title} url={ url }/>);
        expect( container).toMatchSnapshot();
        
     })
 })