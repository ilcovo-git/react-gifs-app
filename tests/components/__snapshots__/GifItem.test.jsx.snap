// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Evalua Componenet <GifItem /> should be match prop title and url at <GifItem /> component with snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="La imagen más bonita"
      src="https://postman"
    />
    <p>
      La imagen más bonita
    </p>
  </div>
</div>
`;
